pontoon/tags/utils/translations.py

```diff
@@ -0,0 +1,50 @@
+
+from django.db.models import Max, Q
+
+from .base import TagsTRTool
+
+
+class TagsTranslationsTool(TagsTRTool):
```

Comment:
Maybe `TagsLatestTranslationsTool`?

Reply:
yep - that probs reflects the purpose better